MediaBrowser.WebDashboard/Api/PackageCreator.cs

```diff
@@ -156,39 +125,28 @@ private string GetCommonJavascript(string mode, string version)
             {
                 builder.AppendFormat("window.appMode='{0}';", mode);
             }
-
             else
             {
                 builder.AppendFormat("window.dashboardVersion='{0}';", version);
             }
 
             builder.Append("</script>");
 
-            var versionString = string.IsNullOrWhiteSpace(mode) ? "?v=" + version : string.Empty;
-
-            var files = new List<string>();
-
-            files.Add("scripts/apploader.js" + versionString);
-
             if (string.Equals(mode, "cordova", StringComparison.OrdinalIgnoreCase))
             {
-                files.Insert(0, "cordova.js");
+                builder.Append("<script src=\"cordova.js\" defer></script>");
```

Comment:
This entire section can be replaced with the web build process which I think is great.